### PR TITLE
WorkFlowDefintion: increase test coverage

### PR DESCRIPTION
**Pull request by eloycoto**

### CODE REVIEW

File: workflow-service/src/main/java/com/redhat/parodos/workflow/definition/service/WorkFlowDefinitionServiceImpl.java [hunk 1]
```diff
@@ -255,6 +256,7 @@ private void getWorksFromWorkDefinition(List<WorkFlowWorkDefinition> workFlowWor
 								workFlowWorkDefinition.getWorkDefinitionId());
 						return;
 					}
+					log.error("Here is a change");
```
Comment (pkliczewski) [inline workflow-service/src/main/java/com/redhat/parodos/workflow/definition/service/WorkFlowDefinitionServiceImpl.java:259]: Is this log line needed?

**Review by pkliczewski — COMMENTED** [review]
Looks good, one comment

File: workflow-service/src/main/java/com/redhat/parodos/workflow/definition/service/WorkFlowDefinitionServiceImpl.java (hunk 1 above)
Comment (eloycoto) [reply workflow-service/src/main/java/com/redhat/parodos/workflow/definition/service/WorkFlowDefinitionServiceImpl.java:259]: clearly no

**Review by pkliczewski — COMMENTED** [review]
/lgtm

**Pull request merged**